README.md

```diff
@@ -1,127 +1,33 @@
-# TODO: Revise this README
+# Digitale Identiteit - IRMA Demo site
 
-# Repository di_node_container
+This project provides the source code of the Amsterdam IRMA demo website.
+It consists of a React frontend and NodeJS backend which is in charge of communicating with an IRMA server. 
+This project does not provide a locally running IRMA server.
 
-## Config
+## Configure
 
-The `CONFIG` environment variable points to a configuration file.
+Make sure to configure the backend properly. By default it is configured to connect to the Amsterdam IRMA server on the Acceptance environment. To be able to connect locally you will need to obtain the PRIVATE key.
+
+Open ```/server/config/config.json```.
 
 ```javascript
 {
-    /* The requestor defined in irmaserver.json in the IRMA server */
-    "requestorname": "openstad_voting_pk",
-
-    /* Every poll must have it's unique uuid, use a generator to create one */
-    "uuid": "06c5a013-4e71-439a-b8da-65e35b6419f0",
+    /* The requestor defined in the configuration of the IRMA server */
+    "requestorname": "irma-demo",
 
     /* URL of the IRMA server */
     "irma": "https://attr.auth.amsterdam.nl",
 
-    /* URL of Node server, optional with port number */
-    "nodeUrl": "http://localhost:8000",
-
     /* Port number the Node server listens to */
     "port": 8000,
 
     /* Document root of site to serve, only for demo */
     "docroot": "../openstad",
```

Comment:
Dit lijkt me nog niet helemaal kloppen

Reply:
jawel, nodeUrl werd helemaal niet gebruikt en in de nieuwe opzet ook niet.